aspnetcore/fundamentals/native-aot.md

```diff
@@ -29,9 +29,9 @@ AOT compilation happens when the app is published. Native AOT is enabled with th
 </PropertyGroup>
 ```
 
-A project that uses native AOT publishing will use JIT compilation when debugging/running, but there are some observable differences:
+A project that uses native AOT publishing uses JIT compilation when running locally. The AOT app has the following differences:
 
-* Some features that aren't compatible with native AOT are disabled and throw exceptions at runtime.
+* Features that aren't compatible with native AOT are disabled and throw exceptions at runtime.
```

Comment:
I think the original statement of "Some features" is probably more accurate. We've tried to make it so that things don't silently fail, but I don't think we are at 100% yet. If we are writing this as it should be for .NET 8.0 RTM then yes, this is fine.